formulas/functions/statistical.js

```diff
@@ -215,7 +215,23 @@ const StatisticalFunctions = {
 
   MAX: () => {},
 
-  MAXA: () => {},
+  MAXA: (...values) => {
+    const arr = [];
+    arr.push(...values.map(a => a.value))
+    const flatArr = FormulaHelpers.flattenDeep(arr)
+    const updatedArr = flatArr.map(a => {
+      if(a === true)
+        return 1
+      if(!isNaN(parseFloat(a)))
+        return parseFloat(a)
+      if(!isNaN(parseInt(a)))
+        return parseInt(a)
+      return 0
+    })
+    if(updatedArr.length === 0)
+      return 0;
+    return Math.max(...updatedArr)
```

Comment:
Getting very "edge-case" here, but spreading an array into `Math.max()` (and similar functions, like `push()` above) can fail if there are too many elements in the array (you can end up hitting the JS parameter limit). 

It's safer to use a loop (or something that loops, like `Array.reduce()`)

```suggestion
    return updatedArr.reduce((a, b) => {
      return Math.max(a, b);
    }, -Infinity);
```

See: 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max

(note, the example they give with `reduce()` ^^ would actually fail if the array contained all negative numbers, since it starts with `0` instead of `-Infinity`.)

Which links to:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply#using_apply_and_built-in_functions

Reply:
Update (sorry these comments are a bit contradictory at this point. I went and ate lunch and couldn't stop thinking about this, haha):

If we use `reduce` above to coerce the `inputArr` to appropriate numbers (and remove `NaN`s), then we should probably do the max comparison in the same loop, rather than running two loops.

Something like:

```js
let numCount = 0;
const max = flatArr.reduce((a, b) => {
  // coerce to number, increment numCount if !isNaN
  
  // compare a to b, return larger
}, -Infinity);
// return max if numCount > 0, else 0 or Error
```

Reply:
Since I filter out all of the nulls earlier, I think I can just run a single reduce loop

Reply:
return finalArr.reduce((a,b) => Math.max(a,b))

Reply:
Also I take care of the case when finalArr is of len 0 earlier, so I dont think a default is nessessary